# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsPIDOMWindow.h
# Commit: 63d9972bab1f
# Full Hash: 63d9972bab1f267a940ec30c7cea7a4c41e54fa6
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-06-06 08:52:03
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing leaks @ mochitest.toml
#   
#   This reverts commit a97cb1d6c08bc38e8644bfc916298b847dbac3a1.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 801e25cb4bfd -r 63d9972bab1f dom/base/nsPIDOMWindow.h
--- a/dom/base/nsPIDOMWindow.h	Thu Jun 05 18:51:36 2025 +0000
+++ b/dom/base/nsPIDOMWindow.h	Thu Jun 05 21:40:49 2025 +0300
@@ -66,7 +66,6 @@
 class ServiceWorkerDescriptor;
 class Timeout;
 class TimeoutManager;
-class WebIdentityHandler;
 class WindowContext;
 class WindowGlobalChild;
 class CustomElementRegistry;
@@ -622,8 +621,6 @@
 
   bool UsingStorageAccess();
 
-  mozilla::dom::WebIdentityHandler* GetOrCreateWebIdentityHandler();
-
   uint32_t UpdateLockCount(bool aIncrement) {
     MOZ_ASSERT_IF(!aIncrement, mLockCount > 0);
     mLockCount += aIncrement ? 1 : -1;
@@ -669,7 +666,6 @@
 
   RefPtr<mozilla::dom::Performance> mPerformance;
   mozilla::UniquePtr<mozilla::dom::TimeoutManager> mTimeoutManager;
-  RefPtr<mozilla::dom::WebIdentityHandler> mWebIdentityHandler;
 
   RefPtr<mozilla::dom::Navigation> mNavigation;
 